src/Main.elm

```diff
@@ -694,15 +676,15 @@ viewFormFieldOptionsPreview { formValues, customAttrs, shortTextTypeDict } formF
             fieldNameOf formField
 
         chosenForYou choices =
-            case ( formField.presence, choices ) of
-                ( SystemRequired _, [ _ ] ) ->
-                    True
+            case formField.presence of
+                Optional ->
+                    False
 
-                ( Required, [ _ ] ) ->
-                    True
+                Required ->
+                    List.length choices == 1
 
-                _ ->
-                    False
+                System ->
+                    List.length choices == 1
     in
```

Comment:
prefer exhaustive matching, i.e. prefer not to have `_ ->`. we can do that by switching fancy `[ _ ]` matching to just `List.length choices == 1`